kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -94,6 +94,18 @@ class KyuubiApplicationManager extends AbstractService("KyuubiApplicationManager
       case None => None
     }
   }
+
+  private def deleteTempDirForUpload(): Unit = {
+    try {
+      FileUtils.deleteDirectory(KyuubiApplicationManager.tempDirForUpload.toFile)
```

Comment:
why not use Utils.deleteDirectoryRecursively

Reply:
Yes, it's now using `Utils.deleteDirectoryRecursively` here.